modules/post/multi/gather/aws_ec2_instance_metadata.rb

```diff
@@ -36,7 +36,7 @@ def initialize(info = {})
 
   def check_aws_metadata
     resp = simple_get(@target_uri)
-    unless resp =~ /^instance-id.$/m
+    unless resp =~ /instance-id/m
```

Comment:
Without context, I'm not sure why this regex needed to be replaced. However, if the change makes sense, then regex is unnecessary here and `resp.to_s.include?('instance-id')` is preferred.